css/MMM-text-clock.css

```diff
@@ -22,6 +22,14 @@
   display: grid;
 }
 
+.MMM-text-clock div.module-content div.tiny {
+  font-size: 15px;
```

Comment:
I'm using MagicMirror's [css classes](https://github.com/MichMich/MagicMirror/blob/master/css/main.css#L62) to set the font size of the clock based on the `size` setting in the config. Although I like `tiny` as a word and config better I think it's easier if it is renamed to `xsmall`. That way this piece of CSS can be removed and font-size will be set automatically.

The `xsmall` class sets the font-size to `1rem` which will convert to 16px in most cases. I see you put the font-size to 15px. Is 16px okay as wel? 

If not I would suggest the following:
Leave the size named `tiny`, change the selector to `.grid--tiny` and add the following to `MMM-text-clock.js`  at line ~224:
```
if(this.size === 'tiny') {
  grid.classList.add('grid--tiny');
}
```

Hope this is clear :)

Reply:
Thanks for a quick response. I didn't know that xsmall exists, I'll convert my changes to use it.

Reply:
Thanks!